src/Uno.UI/UI/Xaml/DependencyProperty.cs

```diff
@@ -72,6 +72,25 @@ private DependencyProperty(string name, Type propertyType, Type ownerType, Prope
 			CachedHashCode = _name.GetHashCode() ^ ownerType.GetHashCode();
 		}
 
+		// This is our equivalent of WinUI's ValidateXXX methods in PropertySystem.cpp, e.g, CDependencyObject::ValidateFloatValue
+		internal void ValidateValue(object value)
+		{
+			if (this == FrameworkElement.MaxWidthProperty ||
+				this == FrameworkElement.MinWidthProperty ||
+				this == FrameworkElement.MaxHeightProperty ||
+				this == FrameworkElement.MinHeightProperty)
```

Comment:
Thinking about it again, we have `_flags` that would avoid making this particular set of checks. Let's use that instead.

Reply:
Let's do that one for now, until we add additional validations.